### DISCUSSION TITLE
ji-tang/info/h7Rsub-yMj71M_YWbrCkg

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#14] [Announcements]
# ji-tang/info/h7Rsub-yMj71M_YWbrCkg

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/ji-tang/info/h7Rsub-yMj71M_YWbrCkg?asd=123

**Comment by wurencaideli** [comment]
哈哈哈
